Documentation/cassandra-operator-upgrade.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Cassandra Operator Upgrade
+weight: 51
+---
+
+# Cassandra Operator Upgrades
+
+This guide will walk you through the manual steps to upgrade the software in Cassandra Operator from one version to the next. The cassandra operator is made up of two parts: 
+1. The main controller that runs as a standalone application, watches the Cassandra Cluster CRD and makes administrative decisions. We'll call this component `Controller` for the rest of this document.
+2. A sidecar that runs alongside each member of a Cassandra Cluster. We will call this component `Sidecar`.
+
+Both components should be updated. This is a very manual process at the moment, but it should be automated soon in the future, once the Cassandra Operator reaches the beta stage.
+
+## Considerations
+With this upgrade guide, there are a few notes to consider:
+* **WARNING:** Upgrading a Rook cluster is not without risk. There may be unexpected issues or
+  obstacles that damage the integrity and health of your storage cluster, including data loss. Only
+  proceed with this guide if you are comfortable with that. It is recommended that you backup your data before proceeding.
+* **WARNING:** The current process to upgrade REQUIRES the cluster to be unavailable for the time of the upgrade.
+
+## Prerequisites
+
+* Before starting the procedure, ensure that your Cassandra Clusters are in a healthy state. You can check a Cassandra Cluster's health by using `kubectl describe $NAME -n $NAMESPACE` and ensuring that for each rack in the Status, `readyMembers` equals `members`.
+* If you are upgrading from v0.9.1 to a later version, the mount point of the PVC for each member has changed because it was wrong. 
```

Comment:
"it was wrong" isn't really correct here from my perspsective.
It was correct and still is, but it was not optimal in point that all persistent and "temporary" (journal, write log or so) data on the volume.